Lesson1/T_Ilya/CPP_ADV_Lesson1/functions.cpp

```diff
@@ -0,0 +1,60 @@
+#include "functions.h"
+#include "task.h"
+
+namespace MFun{
+void print_vect(std::vector<int>& a_vect){
```

Comment:
use const reference if you don't change content of vector:
`const std::vector<int>&`

Reply:
done